content/projects/gsoc/index.adoc

```diff
@@ -41,11 +39,15 @@ We are applying to participate in Google Summer of Code 2023.
 // * link:/projects/gsoc/2022/projects/automatic-git-cache-maintenance[Automatic git cache maintenance on the controller] by Hrushikesh Rao
 // * link:/projects/gsoc/2022/projects/pipeline-step-documentation-generator[Pipeline Step Documentation Generator Improvements] by Vihaan Thora
 
-We are currently collecting project ideas. 
-Add your ideas by submitting an ad-hoc pull request as explained in our latest link:/blog/2022/11/16/gsoc-2023/[GSoC blog post].
+We are currently waiting to know whether we are accepted as a participating organisation (to be announced on Feb 22). 
```

Comment:
```suggestion
We are currently waiting to know whether we will be accepted as a participating organisation (to be announced on Feb 23). 
```

Reply:
Thanks Kris. :+1:

Had definitely a doubt with "whether" and "wether" :-)

Reply:
Thanks for the grammatical correction, Kris.
Although a detail, I'll keep the date as is. I checked the timeline and Google will publish the participating orgs on Feb 22 at 18:00 UTC. ;-)